packages/@aws-cdk/aws-codepipeline/test/pipeline.test.ts

```diff
@@ -566,3 +859,57 @@ class ReusePipelineStack extends cdk.Stack {
     });
   }
 }
+
+const createPipeline = (scope: Construct, pipelineName?: string, pipelineId: string = 'Pipeline') => {
+  const artifact = new codepipeline.Artifact();
+  return new codepipeline.Pipeline(scope, pipelineId, {
+    pipelineName,
+    crossAccountKeys: true,
+    reuseCrossRegionSupportStacks: false,
+    stages: [
+      {
+        stageName: 'Source',
+        actions: [new FakeSourceAction({ actionName: 'Source', output: artifact })],
+      },
+      {
+        stageName: 'Build',
+        actions: [new FakeBuildAction({ actionName: 'Build', input: artifact })],
+      },
+    ],
+  });
+};
+
+interface CreatePipelineStackOptions {
+  context: cdk.App,
+  suffix: string,
+  stackId?: string,
+  pipelineId?: string,
+  undefinedStackName?: boolean,
+  nestedStackId?: string,
+}
+
+const createPipelineStack = (options: CreatePipelineStackOptions): PipelineStack => {
```

Comment:
Is this `createPipelineStack()` helper function really needed? What about just instantiating `PipelineStack` in the tests directly? I feel like using a construct instead of a function makes the tests read more like idiomatic CDK code.

Reply:
For example, this test:

```ts
      const stack = createPipelineStack({
        context: app,
        suffix: 'Name',
        stackId: 'PipelineStack',
      });

      Template.fromStack(stack).hasResourceProperties(kmsAliasResource, {
        AliasName: 'alias/codepipeline-actualstacknamepipeline0a412eb5',
      });
```

I think would read much better if we created the `PipelineStack` directly, instead of using the `createPipelineStack()` helper.